packages/lesswrong/components/search/PostsSearchHit.jsx

```diff
@@ -25,7 +25,7 @@ const PostsSearchHit = ({hit, clickAction, router, classes}) => {
   // If clickAction is provided, disable link and replace with Click of the action
   return <div className={classes.root}>
     <Link
-      onClick={(event) => isLeftClick(event) && clickAction()}
+      onClick={(event) => isLeftClick(event) && clickAction && clickAction()}
```

Comment:
The defensive programming is good, but this is actually pointing to a problem one step up the call stack; in `SearchBarResult` we have `<Components.PostsSearchHit clickAction={this.closeSearch} {...props} />`. This is the only usage of `PostsSearchHit`, and `this.closeSearch` is undefined there. Since that was presumably meant to be doing something, we should figure out what, and fix it there.